mm/2s2h/Rando/Logic/Regions/Moon.cpp

```diff
@@ -108,7 +108,10 @@ static RegisterShipInitFunc initFunc([]() {
                 (RANDO_SAVE_OPTIONS[RO_ACCESS_TRIALS] == RO_ACCESS_TRIALS_FORMS) ||
                 (RANDO_SAVE_OPTIONS[RO_ACCESS_TRIALS] == RO_ACCESS_TRIALS_OPEN)
             ),
-            EXIT(ENTRANCE(MAJORAS_LAIR, 0),                          ONE_WAY_EXIT, RemainsCount() >= RANDO_SAVE_OPTIONS[RO_ACCESS_MAJORA_REMAINS_COUNT]),
+            EXIT(ENTRANCE(MAJORAS_LAIR, 0),                          ONE_WAY_EXIT, 
+                (RemainsCount() >= RANDO_SAVE_OPTIONS[RO_ACCESS_MAJORA_REMAINS_COUNT]) ||
+                (MoonMaskCount() >= RANDO_SAVE_OPTIONS[RO_ACCESS_MAJORA_MASKS_COUNT])
```

Comment:
This area ignores clang format.
```suggestion
            EXIT(ENTRANCE(MAJORAS_LAIR, 0),                          ONE_WAY_EXIT,  (RemainsCount() >= RANDO_SAVE_OPTIONS[RO_ACCESS_MAJORA_REMAINS_COUNT]) ||  (MoonMaskCount() >= RANDO_SAVE_OPTIONS[RO_ACCESS_MAJORA_MASKS_COUNT])
```